File: 16_Behaviour Driver Development/Praktikum/Code program soal prioritas 2/src/test/java/starter/payment/Payment.java
package starter.payment;

import net.thucydides.core.annotations.Step;

public class Payment {
    @Step("I am on the payment page")
    public void onThePaymentPage(){
        System.out.println("I am on the payment page");
    }
    @Step("I choose payment method example Dana")
    public void choosePaymentMethod(){
        System.out.println("I choose payment method example Dana");
    }
    @Step("I click pay button")
    public void clickPayButton(){
        System.out.println("I click pay button");
    }
    @Step("Redirected to Dana application")
    public void redirectToDanaApp(){
        System.out.println("Redirected to Dana application");
    }
}

File: 16_Behaviour Driver Development/Praktikum/Code program soal explorasi/cypress/e2e/Payment/Payment.js
import {Given , When, And, Then} from "cypress-cucumber-preprocessor/steps";

Given ("I am on the payment page",function(){
    console.log("On the payment page")
});
When ("I choose payment method example Dana",function(){
    console.log("Choose payment method")
});
And ("I click pay button",function(){
    console.log("Click pay button")
});
Then ("Redirected to Dana application",function(){
    console.log("Redirect to Dana app")
});
